server/app/assets/stylesheets/uswds/_uswds-theme-custom-styles.scss

```diff
@@ -62,3 +62,38 @@ See the radio-option below as an example.
 .cf-north-star-enumerator-field:last-child {
   @include u-padding-bottom(0)
 }
+
+/**
+ * The USWDS header doesn't give enough space for the logo and title, so we add some.
+ */
+@media (min-width: 64em) {
+  .usa-header--basic .usa-navbar {
+    width: 50%;
+  }
+
+  .usa-header--basic .usa-nav {
+    @include u-padding-top(4);
+    @include u-padding-bottom(2);
+  }
+}
+
+@media (max-width: 63.9em) {
+  .usa-nav ul {
+    margin-bottom: 8px;
+  }
+}
+
+.usa-header--basic .usa-button {
+  @include u-line-height('sans', 2); 
+}
+
+/**
+ * Default form styles add a bottom margin to forms, but we don't want that for the language
+ * selector as it would mess with header styles.
+ */
+
+.usa-header--basic form {
```

Comment:
**Question**:  Should we add a specific class on the language selector rather then removing it from all forms in the header? Or will we never want it on any form in the header?

Reply:
If there was another form in the header (which seems unlikely), we would also want to remove that style so it doesn't mess with the alignment of the header buttons